src/arrayMethodSort.js

```diff
@@ -1,11 +1,30 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const compareFn =
+      compareFunction ||
+      function (a, b) {
+        if (a.toString() > b.toString()) {
+          return 1;
+        }
+
+        if (a.toString() < b.toString()) {
+          return -1;
+        }
+
+        return 0;
```

Comment:
The default compare function uses `toString()` to compare elements, which works for string and number comparisons. However, consider that for objects or other types, this might not yield the expected results. If the task allows, you might want to handle such cases explicitly or document this behavior.